tests/run.sh

```diff
@@ -1,3 +1,13 @@
 #!/bin/sh
 
-nvim --headless -u tests/init.lua -c "PlenaryBustedDirectory tests/ {minimal_init = 'tests/init.lua', sequential = true}"
+# The Tests Runner runs tests for all the environments
+
+mkdir -p ~/.local/share/nvim/site/pack/vendor/start
+git clone --depth 1 https://github.com/nvim-lua/plenary.nvim ~/.local/share/nvim/site/pack/vendor/start/plenary.nvim
+git clone --depth 1 https://github.com/nvim-telescope/telescope.nvim ~/.local/share/nvim/site/pack/vendor/start/telescope.nvim
+
+nvim --headless -u tests/environments/minimal.vim -c "PlenaryBustedDirectory tests/ {minimal_init = 'tests/environments/minimal.vim', sequential = true}"
```

Comment:
The tests runner would run everything, and later we can parametrized to be able to run just a subset